workflows/diagnostics/fv3net/diagnostics/offline/compute.py

```diff
@@ -246,7 +258,11 @@ def _add_derived_diagnostics(ds):
     return merged.assign_attrs(ds.attrs)
 
 
-def _coarsen_transform(evaluation_resolution: int, prediction_resolution: int):
+def _coarsen_transform(
+    evaluation_resolution: int,
+    prediction_resolution: int,
+    catalog: intake.catalog.local.YAMLFilesCatalog,
```

Comment:
I think this should be sufficient for the type hint:
```suggestion
    catalog: intake.Catalog,
```